libcloudforensics/providers/gcp/internal/common.py

```diff
@@ -25,18 +24,19 @@
 from google.auth import default
 from google.auth.exceptions import DefaultCredentialsError, RefreshError
 from googleapiclient.discovery import build
+from libcloudforensics import logging_utils  # pylint: disable=ungrouped-imports
```

Comment:
Why are we disabling the ungrouped imports here? Also, see my other comment about just using the native `logging.getLogger()` function

Reply:
pylint sees the fake import under `if TYPE_CHECKING` so it wants the actual import there as well